### PR TITLE
Bitcoinjs

### PR DESCRIPTION
**Pull request by dl3br**
builder addOutput now takes Buffer | string
nullData.output.encode Buffer
fixing bugs according to @bobrosoft

### CODE REVIEW

**Review by bobrosoft — APPROVED** [review]
Looks correct.

**Comment by dlebrecht** [comment]
@bobrosoft it takes usually around a week for the changes to show up in npm repository

if u want them now, point npm to my github branch

**Comment by bobrosoft** [comment]
@dlebrecht thanks, np. I used `as any` in code for now to overcome that, will replace later as changes show.

File: types/bitcoinjs-lib/index.d.ts
Comment (DanielRosenwasser) [inline types/bitcoinjs-lib/index.d.ts]: Can you extract the element type into its own type?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
Approved by third party. PR appears ready to merge pending review by a maintainer.

**Pull request merged**
